WcaOnRails/app/webpacker/components/UserBadge.js

```diff
@@ -50,8 +50,10 @@ function UserBadge({
   badgeClasses = '',
   senior = false,
   leader = false,
+  hideBorder = false,
+  leftAlign = false,
 }) {
-  const classes = classnames('user-badge', badgeClasses, { senior, leader });
+  const classes = classnames('user-badge', badgeClasses, { senior, leader, leftAlign });
```

Comment:
Our naming convention for CSS classes is `left-align`

Reply:
Done, thanks for pointing this.